src/app.js

```diff
@@ -1,11 +1,28 @@
 const express = require('express');
+const session = require('express-session');
+const passport = require('passport');
 require('dotenv').config();
+
 const Database = require('./db');
+const authRouter = require('./routes/auth');
 
 const app = express();
 
 app.use(express.json());
 app.use(express.urlencoded({ extended: false }));
+app.use(
+  session({
+    secret: process.env.SESSION_SECRET,
+    resave: false,
+    saveUninitialized: false,
+    cookie: {
+      maxAge: 1000 * 3600 * 24,
```

Comment:
Any particular reason for keeping cookie validity as 1 day?

Reply:
Hi Shrreya, there is no reason for that I can change it if you want me to.

Reply:
I wanted to check if you're planning to have a `rememberMe` option similar to the auth assignments in that case the cookie expiry time was extended to 2 weeks?